shell/src/main/java/alluxio/shell/command/LsCommand.java

```diff
@@ -99,26 +99,33 @@ protected Options getOptions() {
     return new Options()
         .addOption(RECURSIVE_OPTION)
         .addOption(FORCE_OPTION)
-        .addOption(LIST_DIR_AS_FILE_OPTION);
+        .addOption(LIST_DIR_AS_FILE_OPTION)
+        .addOption(LIST_PINNED_FILES_OPTION);
   }
 
   /**
    * Displays information for all directories and files directly under the path specified in args.
    *
-   * @param path The {@link AlluxioURI} path as the input of the command
+   * @param path      The {@link AlluxioURI} path as the input of the command
    * @param recursive Whether list the path recursively
    * @param dirAsFile list the directory status as a plain file
    * @throws AlluxioException when Alluxio exception occurs
-   * @throws IOException when non-Alluxio exception occurs
+   * @throws IOException      when non-Alluxio exception occurs
    */
-  private void ls(AlluxioURI path, boolean recursive, boolean forceLoadMetadata, boolean dirAsFile)
-      throws AlluxioException, IOException {
+  private void ls(AlluxioURI path, boolean recursive, boolean forceLoadMetadata, boolean dirAsFile,
```

Comment:
it will be more maintainable to move `recursive`, `forceLoadMetadata`, `dirAsFile`, `pinnedMetadata` to `ListStatusOptions`. also s/pinnedMetadata/pinned, it's simpler and clear.